src/dmd/root/speller.d

```diff
@@ -99,7 +99,7 @@ private auto spellerY(alias dg)(const(char)* seed, size_t seedlen, size_t index,
     return p; // return "best" result
 }
 
-private auto spellerX(alias dg)(const(char)* seed, size_t seedlen, int flag)
+private auto spellerX(alias dg)(const(char)* seed, size_t seedlen, const bool flag)
```

Comment:
`const` on value type is merely noise (and doesn't match the predominant style AFAIK).
`const` on ref type is great - but on value type, it's only showing implementation details of the function, nothing that affects the API.

Reply:
@Geod24 why do we prefer `const` on local variables but not on parameters?

Reply:
I think we already had this conversation ? IMHO it shouldn't be visible in the function's API whether or not it is `const`, because it's none of the other functions' business. But I don't mind too much one way or another.